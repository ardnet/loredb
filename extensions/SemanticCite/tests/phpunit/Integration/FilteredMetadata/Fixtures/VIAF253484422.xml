<?xml version="1.0"  encoding="UTF-8"?> 
<ns2:VIAFCluster xmlns="http://viaf.org/viaf/terms#" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:void="http://rdfs.org/ns/void#" xmlns:ns2="http://viaf.org/viaf/terms#"><ns2:viafID>253484422</ns2:viafID><ns2:Document about="http://viaf.org/viaf/253484422/"><ns2:inDataset resource="http://viaf.org/viaf/data"/><ns2:primaryTopic resource="http://viaf.org/viaf/253484422"/></ns2:Document><ns2:nameType>Personal</ns2:nameType><ns2:sources><ns2:source nsid="00163864">NDL|00163864</ns2:source><ns2:source nsid="nr92005566">LC|nr 92005566</ns2:source><ns2:source nsid="0000000376206155">ISNI|0000000376206155</ns2:source><ns2:source differentiated="false" nsid="http://d-nb.info/gnd/159934044" sparse="true">DNB|159934044</ns2:source></ns2:sources><ns2:length>154</ns2:length><ns2:mainHeadings><ns2:data><ns2:text>&#x624B;&#x675F;, &#x6B63;&#x662D;, 1944-</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data><ns2:data><ns2:text>Tezuka, Masaaki, 1944-</ns2:text><ns2:sources><ns2:s>LC</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data><ns2:data><ns2:text>Tezuka, Masaaki</ns2:text><ns2:sources><ns2:s>DNB</ns2:s></ns2:sources></ns2:data><ns2:mainHeadingEl><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="100"><ns2:subfield code="a">Tezuka, Masaaki</ns2:subfield></ns2:datafield><ns2:sources><ns2:s>DNB</ns2:s></ns2:sources><ns2:id>DNB|159934044</ns2:id><ns2:links><ns2:link><ns2:match type="exact name"/>ISNI|0000000376206155</ns2:link><ns2:link><ns2:match type="exact name"/>ISNI|0000000376206155</ns2:link><ns2:link><ns2:match type="exact name"/>ISNI|0000000376206155</ns2:link></ns2:links></ns2:mainHeadingEl><ns2:mainHeadingEl><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="100"><ns2:subfield code="a">Tezuka, Masaaki</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield></ns2:datafield><ns2:sources><ns2:s>ISNI</ns2:s></ns2:sources><ns2:id>ISNI|0000000376206155</ns2:id><ns2:links><ns2:link><ns2:match type="partial date and publisher"/>NDL|00163864</ns2:link><ns2:link><ns2:match type="partial date and publisher"/>LC|nr 92005566</ns2:link><ns2:link><ns2:match type="exact name"/>DNB|159934044</ns2:link></ns2:links></ns2:mainHeadingEl><ns2:mainHeadingEl><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="100"><ns2:subfield code="a">Tezuka, Masaaki,</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield></ns2:datafield><ns2:sources><ns2:s>LC</ns2:s></ns2:sources><ns2:id>LC|nr 92005566</ns2:id><ns2:links><ns2:link><ns2:match type="forced single date"/>NDL|00163864</ns2:link><ns2:link><ns2:match type="partial date and publisher"/>ISNI|0000000376206155</ns2:link><ns2:link><ns2:match type="partial date and publisher"/>ISNI|0000000376206155</ns2:link><ns2:link><ns2:match type="partial date and publisher"/>ISNI|0000000376206155</ns2:link></ns2:links></ns2:mainHeadingEl><ns2:mainHeadingEl><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="100"><ns2:subfield code="a">&#x624B;&#x675F;, &#x6B63;&#x662D;,</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield></ns2:datafield><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:id>NDL|00163864</ns2:id><ns2:links><ns2:link><ns2:match type="forced single date"/>LC|nr 92005566</ns2:link><ns2:link><ns2:match type="partial date and publisher"/>ISNI|0000000376206155</ns2:link><ns2:link><ns2:match type="partial date and publisher"/>ISNI|0000000376206155</ns2:link><ns2:link><ns2:match type="partial date and publisher"/>ISNI|0000000376206155</ns2:link></ns2:links></ns2:mainHeadingEl></ns2:mainHeadings><ns2:fixed><ns2:gender>u</ns2:gender></ns2:fixed><ns2:x400s><ns2:x400><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="400"><ns2:subfield code="a">Tezuka, Masaaki,</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield><ns2:normalized>tezuka masaaki 1944</ns2:normalized></ns2:datafield><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:x400><ns2:x400><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="400"><ns2:subfield code="a">Tezuka, Masaaki</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield><ns2:normalized>tezuka masaaki 1944</ns2:normalized></ns2:datafield><ns2:sources><ns2:s>ISNI</ns2:s></ns2:sources></ns2:x400><ns2:x400><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="100"><ns2:subfield code="a">Tezuka, Masaki,</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield><ns2:normalized>tezuka masaki 1944</ns2:normalized></ns2:datafield><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:x400><ns2:x400><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="400"><ns2:subfield code="a">Tezuka, Masaki</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield><ns2:normalized>tezuka masaki 1944</ns2:normalized></ns2:datafield><ns2:sources><ns2:s>ISNI</ns2:s></ns2:sources></ns2:x400><ns2:x400><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="100"><ns2:subfield code="a">&#x30C6;&#x30BA;&#x30AB;, &#x30DE;&#x30B5;&#x30A2;&#x30AD;,</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield><ns2:normalized>&#x30C6;&#x30B9;&#x30AB; &#x30DE;&#x30B5;&#x30A2;&#x30AD; 1944</ns2:normalized></ns2:datafield><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:x400><ns2:x400><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="400"><ns2:subfield code="a">&#x30C6;&#x30BA;&#x30AB;, &#x30DE;&#x30B5;&#x30A2;&#x30AD;</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield><ns2:normalized>&#x30C6;&#x30B9;&#x30AB; &#x30DE;&#x30B5;&#x30A2;&#x30AD; 1944</ns2:normalized></ns2:datafield><ns2:sources><ns2:s>ISNI</ns2:s></ns2:sources></ns2:x400><ns2:x400><ns2:datafield dtype="MARC21" ind1="1" ind2=" " tag="400"><ns2:subfield code="a">&#x624B;&#x675F;&#x6B63;&#x662D;,</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield><ns2:normalized>&#x624B;&#x675F;&#x6B63;&#x662D; 1944</ns2:normalized></ns2:datafield><ns2:sources><ns2:s>LC</ns2:s></ns2:sources></ns2:x400><ns2:x400><ns2:datafield dtype="MARC21" ind1="0" ind2=" " tag="400"><ns2:subfield code="a">&#x624B;&#x675F;&#x6B63;&#x662D;</ns2:subfield><ns2:subfield code="d">1944-</ns2:subfield><ns2:normalized>&#x624B;&#x675F;&#x6B63;&#x662D; 1944</ns2:normalized></ns2:datafield><ns2:sources><ns2:s>ISNI</ns2:s></ns2:sources></ns2:x400></ns2:x400s><ns2:coauthors><ns2:data count="3" tag="951"><ns2:text>&#x9AD8;&#x7802;&#x6559;&#x4F1A;&#x4F1A;&#x5802;&#x5EFA;&#x8A2D;&#x59D4;&#x54E1;&#x4F1A;</ns2:text><ns2:sources><ns2:s>LC</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data><ns2:data count="3" tag="951"><ns2:text>Takasago Kyo&#x304;kai Kaido&#x304; Kensetsu Iinkai.</ns2:text><ns2:sources><ns2:s>LC</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data><ns2:data count="2" tag="950"><ns2:text>&#x8FBB;, &#x7BC0;&#x5B50;</ns2:text><ns2:sources><ns2:s>NDL</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data><ns2:data count="2" tag="950"><ns2:text>Tsuji, Setsuko.</ns2:text><ns2:sources><ns2:s>NDL</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data><ns2:data count="2" tag="950"><ns2:text>Carey, George.</ns2:text><ns2:sources><ns2:s>NDL</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data></ns2:coauthors><ns2:publishers><ns2:data count="10"><ns2:text>&#x30AD;&#x30EA;&#x30B9;&#x30C8;&#x65B0;&#x805E;&#x793E;</ns2:text><ns2:sources><ns2:s>LC</ns2:s><ns2:s>NDL</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data><ns2:data count="5"><ns2:text>&#x30DE;&#x30EB;&#x30B3;&#x30FC;&#x30B7;&#x30E5;&#x30FB;&#x30D1;&#x30D6;&#x30EA;&#x30B1;&#x30FC;&#x30B7;&#x30E7;&#x30F3;</ns2:text><ns2:sources><ns2:s>NDL</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data><ns2:data count="4"><ns2:text>Kirisutoshinbunsha.</ns2:text><ns2:sources><ns2:s>LC</ns2:s><ns2:s>NDL</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data><ns2:data count="2"><ns2:text>&#x751F;&#x3051;&#x308B;&#x6C34;&#x306E;&#x5DDD;</ns2:text><ns2:sources><ns2:s>NDL</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data><ns2:data count="2"><ns2:text>Ikeru Mizu No Kawa</ns2:text><ns2:sources><ns2:s>NDL</ns2:s><ns2:s>ISNI</ns2:s></ns2:sources></ns2:data></ns2:publishers><ns2:birthDate>1944</ns2:birthDate><ns2:deathDate>0</ns2:deathDate><ns2:dateType>lived</ns2:dateType><ns2:dates max="200" min="198"><ns2:date count="3" scaled="1.58">198</ns2:date><ns2:date count="10" scaled="3.32">199</ns2:date><ns2:date count="4" scaled="2.00">200</ns2:date></ns2:dates><ns2:RecFormats><ns2:data count="12"><ns2:text>am</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data><ns2:data count="1"><ns2:text>dm</ns2:text><ns2:sources><ns2:s>LC</ns2:s></ns2:sources></ns2:data></ns2:RecFormats><ns2:ISBNs unique="11"><ns2:data count="2"><ns2:text>9784873952154</ns2:text><ns2:sources><ns2:s>NDL</ns2:s><ns2:s>LC</ns2:s></ns2:sources></ns2:data><ns2:data count="1"><ns2:text>9784873956022</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data><ns2:data count="1"><ns2:text>9784873952840</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data><ns2:data count="1"><ns2:text>9784873952567</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data><ns2:data count="1"><ns2:text>9784873952314</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data><ns2:data count="1"><ns2:text>9784873952055</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data><ns2:data count="1"><ns2:text>9784872072211</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data><ns2:data count="1"><ns2:text>9784872072198</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data><ns2:data count="1"><ns2:text>9784872072150</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data><ns2:data count="1"><ns2:text>9784872071818</ns2:text><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources></ns2:data></ns2:ISBNs><ns2:countries><ns2:data count="13" scaled="4"><ns2:text>JP</ns2:text><ns2:sources><ns2:s>NDL</ns2:s><ns2:s>LC</ns2:s></ns2:sources></ns2:data></ns2:countries><ns2:titles><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>Anata wa yarinaosu koto ga dekiru</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>church in the market place.</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>Hiroba no naka no kyo&#x304;kai</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>Inochi no shu&#x304;kyo&#x304; no kaifuku</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>Kagayakashii kisetsu no hajimari</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>Kirisutokyo&#x304; no daisan no nami.</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>Kiyo&#x304;kai seichiyo&#x304; no kandokoro</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>Seinaru tabi : Kirisutokyo no aratana kanosei o saguru.</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s><ns2:s>LC</ns2:s></ns2:sources><ns2:title>Shinko&#x304; no hyo&#x304;gen to shiteno kyo&#x304;kai kenchiku</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>Yoshua no gotoku ikin</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>&#x3042;&#x306A;&#x305F;&#x306F;&#x3084;&#x308A;&#x76F4;&#x3059;&#x3053;&#x3068;&#x304C;&#x3067;&#x304D;&#x308B;</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>&#x30AD;&#x30EA;&#x30B9;&#x30C8;&#x6559;&#x306E;&#x7B2C;&#x4E09;&#x306E;&#x6CE2;.</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>&#x30E8;&#x30B7;&#x30E5;&#x30A2;&#x306E;&#x5982;&#x304F;&#x751F;&#x304D;&#x3093;</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s><ns2:s>LC</ns2:s></ns2:sources><ns2:title>&#x4FE1;&#x4EF0;&#x306E;&#x8868;&#x73FE;&#x3068;&#x3057;&#x3066;&#x306E;&#x6559;&#x4F1A;&#x5EFA;&#x7BC9; : &#x305D;&#x306E;&#x5927;&#x3044;&#x306A;&#x308B;&#x8A18;&#x9304;</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>&#x8056;&#x306A;&#x308B;&#x65C5; : &#x30AD;&#x30EA;&#x30B9;&#x30C8;&#x6559;&#x306E;&#x672C;&#x8CEA;&#x3092;&#x6289;&#x308B;</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>&#x5E83;&#x5834;&#x306E;&#x4E2D;&#x306E;&#x6559;&#x4F1A;</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>&#x6559;&#x4F1A;&#x6210;&#x9577;&#x306E;&#x52D8;&#x6240;</ns2:title></ns2:work><ns2:work><ns2:sources><ns2:s>NDL</ns2:s></ns2:sources><ns2:title>&#x8F1D;&#x304B;&#x3057;&#x3044;&#x5B63;&#x7BC0;&#x306E;&#x59CB;&#x307E;&#x308A;</ns2:title></ns2:work></ns2:titles><ns2:history><ns2:ht recid="NDL|00163864" time="2012-09-21T12:25:09+00:00" type="add"/><ns2:ht recid="LC|nr 92005566" time="2012-11-26T09:56:13+00:00" type="add"/><ns2:ht recid="ISNI|0000000376206155" time="2013-10-13T19:17:13+00:00" type="add"/><ns2:ht recid="DNB|159934044" time="2014-01-15T05:38:27+00:00" type="add"/></ns2:history></ns2:VIAFCluster>