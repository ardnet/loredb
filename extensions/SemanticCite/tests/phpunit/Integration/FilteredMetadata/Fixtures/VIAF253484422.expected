{{#scite:VIAF253484422
 |type=Personal
 |viaf=253484422
 |ndl=00163864
 |lc=nr92005566
 |isni=0000000376206155
 |dnb=159934044
 |name=手束, 正昭, 1944-
}}